jetty-openid/src/main/config/modules/openid/openid-baseloginservice.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+<Configure id="BaseLoginService">
+  <!-- Optional code to configure the base LoginService used by the OpenIdLoginService
```

Comment:
This is really sub-optimal. Is there another way that a user can enable say the hash login service as a module, have the openid module depend on just login module, and find the existing bean for the login service and set it on the OpenIdLoginService here?

Reply:
Not really sure what you mean here, the openid module only currently depends on the security module. This xml is only if you want to add a base loginservice to the OpenIdLoginService to get user roles, which is entirely optional, and it could be any type of login service not just the HashLoginService.

Reply:
I understand that it can wrap any of the login services. That's why it would be best if we have a module to enable each one of the flavours of our login services and the user can pick any of them to enable. Each of those login service modules would "provide loginservice" so the openid module would just depend on "loginservice". All of the login service modules would have to configure a LoginService with the same, well-known id (eg "loginService") so that the open id module will be able to reference that id in its wrapping. 

I'd prefer it if we could avoid making the user edit a special xml file for this, just use the module mechanism and properties.

Reply:
Currently there are no modules for any of the login services so this change is better to be done in another PR. 
Opened issue #4073.